user_input/input.py

```diff
@@ -9,6 +9,7 @@
 =================================================
 """
 
```

Comment:
Should I just go ahead and encapsulate these functions into an Input class?

Reply:
I'd say if you feel like it would benefit the project, go for it.